person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
- [ ] you are creating the instance variable `@id` and are adding the value of `id` however id is not passed as an argument, what you can do instead is assigning a random number to the instance variable `@id`